gorgone/gorgone/modules/core/proxy/class.pm

```diff
@@ -497,7 +500,10 @@ sub periodic_exec {
                 token => $connector->generate_token(),
                 target => ''
             });
-            $connector->{clients}->{$_}->{class}->close() if (defined($connector->{clients}->{$_}->{class}));
+            if (defined($connector->{clients}->{$_}->{class})) {
```

Comment:
The indentation looks broken here

Reply:
I checked but I don't see any problem here